portfolio/src/main/java/com/google/sps/servlets/MessageFormServlet.java

```diff
@@ -0,0 +1,37 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/message-form-handler")
+public class MessageFormServlet extends HttpServlet {
+
+    @Override
+    public void doPost(final HttpServletRequest request, final HttpServletResponse response) throws IOException {
+
+        // Get the value entered in the form.
+        final String nameValue = request.getParameter("name-input");
+        final String emailValue = request.getParameter("email-input");
+        final String topicValue = request.getParameter("topic-input");
+        final String messageValue = request.getParameter("message-input");
+        
+        if (!nameValue.equals("") && !emailValue.equals("") && !topicValue.equals("") && !messageValue .equals("") ) {
```

Comment:
nit: Prefer using `isEmpty` to be more readable: 

https://www.javatpoint.com/java-string-isempty